source/common/grpc/common.h

```diff
@@ -28,12 +28,47 @@ class Common {
   /**
    * Charge a success/failure stat to a cluster/service/method.
    * @param cluster supplies the target cluster.
+   * @param protocol supplies the downstream protocol in use, either gRPC or gRPC-Web.
```

Comment:
OK, I will keep http1bridge filter unchanged. And use "grpc-web" for grpc_web_filter.